.github/workflows/php-tests.yml

```diff
@@ -43,3 +44,6 @@ jobs:
 
       - name: Run sample
         run: php sample_push.php | grep "Trying tls://gateway.sandbox.push.apple.com:2195"
+
+      - name: Run PHPUnit
```

Comment:
https://github.com/marketplace/actions/setup-php-action#phpunit this needs to be defined before